src/modules/Bootloader.rb

```diff
@@ -90,16 +90,18 @@ def Export
     # @param [Hash] data map of bootloader settings
     # @return [Boolean] true on success
     def Import(data)
-      Yast::BootStorage.detect_disks
+      # AutoYaST configuration mode. There is no access to the system
+      Yast::BootStorage.detect_disks unless Mode.config
```

Comment:
The return value is quite undefined and will be be evaluated by any call. The docu says:
 # Sets properly boot, root and mbr disk.